### ISSUE TITLE
Step failing due to go.mod errors

### ISSUE THREAD

**Issue by csmith** [bitrise-steplib/steps-github-status#29]
This failure seems to have started happening today:

```
+------------------------------------------------------------------------------+

| (2) Update Github status                                                     |
+------------------------------------------------------------------------------+
| id: github-status                                                            |
| version: 2.3.4                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: bash                                                                |
| time: 2021-06-23T14:55:13Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
/tmp/tmp.Cib1j1MiMy/src/github.com/bitrise-steplib/steps-github-status/main.go:12:2: no required module provides package github.com/bitrise-io/go-utils/log: go.mod file not found in current directory or any parent directory; see 'go help modules'
/tmp/tmp.Cib1j1MiMy/src/github.com/bitrise-steplib/steps-github-status/main.go:13:2: no required module provides package github.com/bitrise-tools/go-steputils/stepconf: go.mod file not found in current directory or any parent directory; see 'go help modules'
|                                                                              |
+---+---------------------------------------------------------------+----------+
| ! | Update Github status (exit code: 1)                           | 1.90 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-steplib/steps-github-status/issues |
| Source: https://github.com/bitrise-steplib/steps-github-status               |
+---+---------------------------------------------------------------+----------+
```

It looks like the step is now being run with Go 1.16, which [enables module aware mode by default](https://golang.org/doc/go1.16#go-command), but this repo doesn't have a go.mod file and doesn't set `GO111MODULE` to make Go use the old behaviour.

**Comment by mateherber** [comment]
Hi @csmith, thank you for reporting this. Can you please tell us which stack are you using?

**Comment by csmith** [comment]
Hey @mateherber, it's:

> Stack used:
> Android & Docker, on Ubuntu 20.04

**Comment by lpusok** [comment]
Found that the Ubuntu stacks have a misconfigured GO111MODULE env. That fix is ongoing, in the meantime we released a new GitHub status Step (2.4.0) that is migrated to go modules.

**Comment by csmith** [comment]
Super, thanks for the quick turnaround @lpusok and @mateherber.
